server/container_remove.go

```diff
@@ -23,7 +23,7 @@ func (s *Server) RemoveContainer(ctx context.Context, req *pb.RemoveContainerReq
 	}
 
 	cState := s.runtime.ContainerStatus(c)
-	if cState.Status == oci.ContainerStateCreated || cState.Status == oci.ContainerStateRunning {
+	if cState.Status == oci.ContainerStateRunning {
```

Comment:
@mrunalp this should not happen. In the case you take as example, I expect a `sandbox_remove` to trigger the removal of all containers that are still in created state.
Am I missing something ?